_data/simple-icons.json

```
@@ -6909,6 +6909,11 @@
             "source": "https://kdenlive.org/en/logo/",
             "guidelines": "https://kdenlive.org/en/logo/"
         },
+        {
+            "title": "Kedro",
+            "hex": "FFC900",
+            "source": "https://github.com/kedro-org/kedro-brand-identity/blob/main/icon/black/kedro-icon-svgo-black.svg"
```

```suggestion
            "source": "https://github.com/kedro-org/kedro-brand-identity/blob/7f7b380cb1a2951c06ca292f0d2b442db895f804/icon/black/kedro-icon-svgo-black.svg"
```
We use Permalink when linking to Github files. You can press `y` on your keyboard while viewing a file on Github to get it.